yivi-docs/blog/2025-06-02-age-verification/index.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: 2025-age-verification
+title: Age Verification in the EU, Ambitious goals, missed privacy opportunities
+authors: [dibranmulder]
+tags: [yivi, eudi, age-verification]
+---
+
+## Protecting minors, today and tomorrow
+[The European Commission is preparing a temporary age verification app](https://digital-strategy.ec.europa.eu/en/policies/eu-age-verification#:~:text=This%20initiative%20aims%20to%20allow,pornography%2C%20gambling%2C%20alcohol%2C%20and%20others), aimed at protecting minors on platforms where a minimum age is required—think of online pornography, gambling, alcohol sales, and certain social media services. The need is hardly disputed: in a digital world, it's legitimate to shield minors from harmful content. The Digital Services Act (DSA) already obligates large platforms to take protective measures, and the Commission is stepping in where compliance is lacking. A technical solution to verify age online is therefore urgently needed.
```

Comment:
I don't think the need for an age verification method or app is "hardly disputed". On the contrary, it's very easy to find dissenting opinions from major digital rights organizations like the [EFF](https://www.eff.org/deeplinks/2025/04/age-verification-european-union-mini-id-wallet) and [EDRI](https://edri.org/our-work/policy-paper-age-verification-cant-childproof-the-internet/). Of course "to shield minors from harmful content" _is_ hardly disputed, but this paragraph falsely conflates the two.